client/my-sites/subscribers/components/add-subscribers-modal/add-subscribers-modal.tsx

```diff
@@ -20,10 +20,22 @@ const AddSubscribersModal = ( { site }: AddSubscribersModalProps ) => {
 		useSubscribersPage();
 
 	useEffect( () => {
-		// Open "add subscribers" modal by default via URL
-		if ( window.location.hash === '#add-subscribers' ) {
-			setShowAddSubscribersModal( true );
-		}
+		const handleHashChange = () => {
+			// Open "add subscribers" via URL hash
+			if ( window.location.hash === '#add-subscribers' ) {
+				setShowAddSubscribersModal( true );
+			}
+		};
+
+		// Listen to the hashchange event
+		window.addEventListener( 'hashchange', handleHashChange );
```

Comment:
In the current implementation, the hash persists on the URL after the modal is closed and as a side-effect, the modal is reopened on refresh. So to avoid this, the hash could be cleared when closing the modal.

Reply:
Agreed. I will fix this in the morning. Thanks for the review 🙏